File: Leetcode/PY/Leetcode448-找到所有数组中消失的数字.py
from typing import List
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        n = len(nums)
        for num in nums:
            x = (num - 1) % n
            nums[x] += n
        ans = []
        for i in range(n):
            if nums[i] <= n:
                ans.append(i+1)
        return ans

File: Leetcode/sbgo/Leetcode448-找到所有数组中消失的数字.go
package sbgo

func findDisappearedNumbers(nums []int) []int {
	n := len(nums)
	for _, val := range nums {
		x := (val - 1) % n
		nums[x] += n
	}
	ans := []int{}
	for i := range nums {
		if nums[i] <= n {
			ans = append(ans, i+1)
		}
	}
	return ans
}
